### DISCUSSION TITLE
zig 新计划

### DISCUSSION THREAD

**Discussion by kscooo** [zigcc/forum#33] [General]
原文链接：https://kristoff.it/blog/zig-self-hosted-now-what/

- 性能改进
- 自定义后端
- 新的 for 语法
- 官方包管理器

同样，如果我们想认真对待创建一个可以支持任意大型代码库的亚毫秒增量重建的编译器的目标，我们需要可靠的基础设施。

**Comment by jiacai2050** [comment]
标题有些问题，算是新进展吧……

期待 0.10 版本！

**Reply by kscooo** [reply]
大佬 zig doc 有汉化的计划么

**Comment by jiacai2050** [comment]
现在意义不大，随时都有改动，感觉可以再等等。

**Reply by kscooo** [reply]
有道理，毕竟还没 1.0
